docs/src/catalyst_functionality/compositional_modeling.md

```diff
@@ -91,19 +91,20 @@ ModelingToolkit.get_systems(flatrn)
 More about ModelingToolkit's interface for compositional modeling can be found
 in the [ModelingToolkit docs](http://docs.sciml.ai/ModelingToolkit/stable/).
 
-## Compositional Model of the Repressilator
+## Compositional model of the repressilator
 Let's apply the tooling we've just seen to create the repressilator in a more
 modular fashion. We start by defining a function that creates a negatively
 repressed gene, taking the repressor as input
 ```@example ex1
 function repressed_gene(; R, name)
-    @reaction_network $name begin
-        hillr($R,α,K,n), ∅ --> m
-        (δ,γ), m <--> ∅
-        β, m --> m + P
-        μ, P --> ∅
-    end
+  @reaction_network $name begin
```

Comment:
I am all for 4 spaces, but it a lot of places we use 2 for the macro.

The reason I changed this was that  there was like 6 or 8 spaces for the reactions, and then 4 spaces before the end statement. Mostly wanted to remove indentations for the ned statement. If you want to make it so that 4 spaces are used everwhere that is probably what I prefer as well, just presumed it was 2 spaces because it was os prevalent.